<error errorId="656881e5-0e4c-4beb-9b23-d38ed9ad5d21" host="DESKTOP-OCJNO3J" type="System.Threading.Tasks.TaskCanceledException" message="A task was canceled." source="mscorlib" detail="System.Threading.Tasks.TaskCanceledException: A task was canceled.&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()&#xD;&#xA;   at Petzey.Backend.Appointment.API.Controllers.DashboardController.&lt;GetPetAppointmentsWithFilters&gt;d__4.MoveNext()" time="2024-05-05T17:26:34.5874796Z" />